src/Sarif/CodeGenHints.json

```diff
@@ -656,6 +656,14 @@
           "unicodeCodePoints"
         ]
       }
+    },
+    {
+      "kind": "AttributeHint",
```

Comment:
>AttributeHint [](start = 15, length = 13)

Oops. test gap closed here by adding columnKind explicitly to our comprehensive test files.